kustomize/traffic/namespaceScoped/statefulset.yaml

```diff
@@ -38,8 +38,12 @@ spec:
             memory: 64Mi
         securityContext:
           readOnlyRootFilesystem: true
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - ALL
           runAsNonRoot: true
-          runAsUser: 1000
+          runAsUser: 1001040000
```

Comment:
Could you please provide a reference for this? Where did we pick this value from?

Reply:
I used the value used by the model mesh pods. I have no other reference. Initially I got an error message that the value 1000 was not allowed (low numbers invalid on openshift)